### DISCUSSION TITLE
Is there a way to align figure captions when put side by side in PDF?

### DISCUSSION THREAD

**Discussion by Minh-AnhHuynh** [quarto-dev/quarto-cli#11314] [Q&A] [answered]
### Description

I'm looking forward to replicate something similar to a latex code using \figure and \minipage to put 2 figures side by side.

It seems to me that the latex code aligns the caption by default and not the figure.

Here is a minimal reproducible code:
````qmd
---
title: "Align captions"
format:
  pdf:
    pdf-engine: lualatex
    lof: true
    colorlinks: false
keep-tex: true
---
Reference check to @fig-1 and @fig-2.

::: {layout-ncol=2}
![Caption for Figure 1](figures/figure1.jpg){#fig-1 fig-scap="Short caption for Figure 1"}

![Caption for Figure 2](figures/figure2.jpg){#fig-2 fig-scap="Short caption for Figure 2"}
:::

\begin{figure}[h!]
	\begin{minipage}[b]{0.45\textwidth}
		\centering
		\includegraphics[width=\textwidth]{figures/figure1.jpg}
		\caption[Short caption 1 via latex]{Caption for Figure 1}
		\label{fig:figure1}
	\end{minipage}
	\hfill
	\begin{minipage}[b]{0.45\textwidth}
		\centering
		\includegraphics[width=\textwidth]{figures/figure2.jpg}
		\caption[Short caption 2 via latex]{Caption for Figure 2}
		\label{fig:figure2}
	\end{minipage}
\end{figure}
````
[align-caption.zip](https://github.com/user-attachments/files/17633502/align-caption.zip)

**Comment by mcanouil** [comment, marked as answer]
See https://quarto.org/docs/authoring/figures.html#vertical-alignment

Note that your example is not reproducible.
Also, you should not have bare raw code, use raw code blocks, see https://quarto.org/docs/authoring/markdown-basics.html#raw-content

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by Minh-AnhHuynh** [reply]
My apologies, it seems I have missed this information in that page. Thank you for the informations!
